### DISCUSSION TITLE
Cover Page

### DISCUSSION THREAD

**Discussion by Marta-Paz** [joaomlourenco/novathesis#244] [Q&A] [answered]
Estou a utilizar a versão 6.5.2

Na capa está a surgir "Dissertação para obtenção do grau de mestre em mestrado em engenharia informática". Eu consigo alterar a parte do Engenharia informática. Mas como posso remover o mestrado após o mestre?

**Comment by joaomlourenco** [comment]
Please use english in this forum!

Please upgrade to the latest version (https://github.com/joaomlourenco/novathesis/wiki/upgrading-novathesis#how-to-convert-from-v6x-to-a-more-recent-v6x) and add another comment saying if the problem was solved with the upgrade or persists.

**Comment by Marta-Paz** [comment]
Good morning,

I uptdated to the newly version (6.6.1) but the problem presist. On the cover instead of showing "Dissertação para obtenção do grau de mestre em engenharia informática" it shows "Dissertação para obtenção do grau de mestre em mestrado em engenharia informática" when using the nova/fct-legacy cover

**Comment by joaomlourenco** [comment, marked as answer]
Please upload a copy of your cover…

**Reply by Marta-Paz** [reply]
![Captura de ecrã 2021-11-11 141254](https://user-images.githubusercontent.com/48595397/141312688-e6da2063-866a-4ad4-9192-e128df3c8c03.png)

**Reply by Marta-Paz** [reply]
I do not know if this is the right solution or not, but I found a workaround. On the file strings-pt we have \mscstring(pt):={Mestrado em}. If we take out the 'Mestrado em', it shows what I was expecting

**Reply by joaomlourenco** [reply]
This MSc cover is not allowed anymore.  You have to use the new covers.  Template version 6.5.3 should be using the new covers already.  Something is wrong here…

How is the school defined in "1_novathesis.tex"?    It should be
```latex
\ntsetup{school=nova/fct}
```

**Reply by Marta-Paz** [reply]
I am using:

`\ntsetup{school=nova/fct-legacy}`

**Reply by b-pereira** [reply]
## Answer

> I am using:
> 
> `\ntsetup{school=nova/fct-legacy}`

As far I know NOVA University has new covers. As @joaomlourenco said, that cover isn't allowed anymore and the option `nova/fct-legacy` is self-evident. That is, it's the legacy version. So, to use the current version must use:

```latex
\ntsetup{school=nova/fct}
```
So, to answer completely to your question, this template has one file, to put it grossly, for every cover and schools' standards. So `fct` and `fct-legacy` are two completely different files, with different configurations, as shown in the picture below.

![imagem](https://user-images.githubusercontent.com/944511/141503943-8157932c-4ddc-41ab-b287-1d6f80fb18c6.png)

What you did:

> On the file strings-pt we have \mscstring(pt):={Mestrado em}. If we take out the 'Mestrado em', it shows what I was expecting

was to change **the global string** that **might be** redefined in the `Schools` folder for every school, but it can be redefined in another file. Therefore there can be conflicts. That's **maybe** why taking out the _"Mestrado em"_ will **_"work"_** for a different school configuration.

# Important 
**Also, the _Discussions_ page with the Q&A, as the option to mark as _Answered_**. Given that we don't know if the answer was, in fact, answered, the person that make the question should **mark it as Answered**. This is because the unanswered Questions in Q&A will pile up, and then we have to close it not knowing if it was answered.  @Marta-Paz if you could be so kind, and @joaomlourenco reply is the answer to your question, please do **mark his answer as _Answered_**. Thank you.
